pysrc/bytewax/exhash.py

```diff
@@ -0,0 +1,83 @@
+"""Implementation of a consistent hash for "exchanging" data. Hence
+"exhash".
+
+Bytewax uses this to ensure that items are routed to workers correctly
+and consistently. We cannot use Python's `__hash__` because is not
+consistent across processes by default.
```

Comment:
```suggestion
consistent across processes.
```

Reply:
I wanted to insinuate that we know about `PYTHONHASHSEED` but selected not to use it. But let me write that out explicitly.